x-pack/plugins/observability/public/pages/cases/cases.tsx

```diff
@@ -13,14 +13,15 @@ import { usePluginContext } from '../../hooks/use_plugin_context';
 import { LazyAlertsFlyout } from '../..';
 import { useFetchAlertDetail } from './use_fetch_alert_detail';
 import { useFetchAlertData } from './use_fetch_alert_data';
+import { paths } from '../../config';
 
 interface CasesProps {
   userCanCrud: boolean;
 }
 export const Cases = React.memo<CasesProps>(({ userCanCrud }) => {
   const {
     cases,
-    application: { getUrlForApp, navigateToApp },
```

Comment:
I think we still need to use `navigateToApp`. When I'm testing the link to the Rule Details page was broken because the link misses `server.basePath` which is `eyg` in this link -> `http://localhost:5601/eyg/app/observability/alerts/rules/70c67b70-fb89-11ec-b294-f19be2304125`

Reply:
The `eyg` portion of the URL is the `basePath`. When working with Kibana in "developer mode", a random `basePath` is created to ensure that developers think about the `basePath` when working with URL's.

Reply:
BTW... see the other comment I made about using `http.basePath.prepend` to add the `basePath` to the `linkToRule` URL.